ionic-stencil-essential/index.html

```diff
@@ -71,28 +71,52 @@ <h3>Ionic 3.x</h3>
 </ul>
 <h3>Ionic 4.x</h3>
 <h4>Essential</h4>
-<h5 id="essential-light">Light Mode</h5>
+<h5>Light Mode</h5>
 <ul>
 <li><a href="http://appseed.io/apk-downloads#ionic-stencil-essential">Android APK</a></li>
 <li><a href="https://skounis.github.io/ionic-demo-stencilessential/light/">Web preview</a></li>
 </ul>
-<h5 id="essential-dark">Dark Mode</h5>
+<h5>Dark Mode</h5>
 <ul>
 <li><a href="http://appseed.io/apk-downloads#ionic-stencil-essential">Android APK</a></li>
 <li><a href="https://skounis.github.io/ionic-demo-stencilessential/dark/">Web preview</a></li>
 </ul>
 
 <h4>Awesome</h4>
-<h5 id="awesome-light">Light Mode</h5>
+<h5>Light Mode</h5>
 <ul>
 <li><a href="http://appseed.io/apk-downloads#ionic-stencil-essential" target="_blank">Android APK</a></li>
 <li><a href="https://skounis.github.io/ionic-demo-stencilessential/awesome-theme/light/" target="_blank">Web preview</a></li>
 </ul>
-<h5 id="awesome-dark">Dark Mode</h5>
+<h5>Dark Mode</h5>
 <ul>
 <li><a href="http://appseed.io/apk-downloads#ionic-stencil-essential" target="_blank">Android APK</a></li>
 <li><a href="https://skounis.github.io/ionic-demo-stencilessential/awesome-theme/dark/" target="_blank">Web preview</a></li>
 </ul>
+<h3>Ionic 5.x</h3>
```

Comment:
@orfeasr Let's put "Ionic 5" demos at the top of this section since it is the latest version. The "Ionic 4" and "Ionic 3" version will follow.